src/ApolloClient/client.js

```diff
@@ -1,8 +1,11 @@
 import {ApolloClient, InMemoryCache} from '@apollo/client';
-const client = new ApolloClient
-({
-  uri: 'https://fff-be-2e7913919a6b.herokuapp.com/graphql',
-  // uri: 'http://127.0.0.1:6969/graphql',
+
+const uri = process.env.NODE_ENV === 'production'
```

Comment:
In JavaScript, const is a keyword used to declare a constant variable. Constants are block-scoped, meaning they are accessible only within the block of code where they are defined. Once a value is assigned to a constant using const, it cannot be reassigned. Here's an example: